jax/_src/scipy/signal.py

```diff
@@ -34,6 +34,59 @@
 from jax._src.util import canonicalize_axis, tuple_delete, tuple_insert
 
 
+from scipy.fft import next_fast_len as osp_fft_next_fast_len
```

Comment:
Nit: this import should go next to the other scipy imports

Reply:
done